ui_src/src/domain/schema/components/createSchema/index.js

```diff
@@ -73,15 +74,14 @@ const schemaTypes = [
     },
     {
         id: 4,
-        value: 'avro',
-        label: 'Avro (Coming soon)',
+        value: 'Avro',
+        label: 'Avro (New)',
         description: (
             <span>
                 The popular. Apache Avro™ is the leading serialization format for record data, and first choice for streaming data pipelines. It offers excellent schema
```

Comment:
> Add a note saying that at the moment it is only available here to create schemas but they are not enforced yet on the SDK. Also, make sure that in case of an Avro schema attached to a station not causing issues for connected clients or for clients who trying to produce/consume data

I can't find any better way to prevent connected clients error out if avro schema attached to the station. 
Can we update all the SDKs and merge it all together? I'm all ears if you have any suggestion.
I already worked on Go, JS, Python SDKs. I can create pull request today or tomorrow.

Reply:
So sure please open the PRs on the SDKs and we will merge this one only after it won't crash clients

Reply:
After merging the PRs of the SDKs please also update the readme file on this repo, you will find there a sdk features table

Reply:
> So sure please open the PRs on the SDKs and we will merge this one only after it won't crash clients

Sweet as. Just to let you know I'm only familiar with the above mentioned languages. If somebody can work on the remaining SDKs(Java, .Net, and Rust), that would be great.

Reply:
> After merging the PRs of the SDKs please also update the readme file on this repo, you will find there a sdk features table

Yep. Sure.

Reply:
> > So sure please open the PRs on the SDKs and we will merge this one only after it won't crash clients
> 
> Sweet as. Just to let you know I'm only familiar with the above mentioned languages. If somebody can work on the remaining SDKs(Java, .Net, and Rust), that would be great.

Sure would you mind to open issues on those repositories?

Reply:
> 

Yep. I'm on it.

Reply:
> > > So sure please open the PRs on the SDKs and we will merge this one only after it won't crash clients
> > 
> > 
> > Sweet as. Just to let you know I'm only familiar with the above mentioned languages. If somebody can work on the remaining SDKs(Java, .Net, and Rust), that would be great.
> 
> Sure would you mind to open issues on those repositories?

Issues opened:
https://github.com/memphisdev/memphis.java/issues/42
https://github.com/memphisdev/memphis.net/issues/105

For Rust, there's already an open issue. Not sure i need to open another one.
https://github.com/turulix/memphis-rust-community/issues/15